src/main/java/org/fungover/storm/client/ClientHandler.java

```diff
@@ -32,7 +33,11 @@ public void run() {
             out.close();
             clientSocket.close();
         } catch (IOException e) {
-            LOGGER.error(e.getMessage());
+            statusCode = new HttpResponseStatusCodes();
```

Comment:
Could there be a check here in the future to see if the connection still is open and no data has ben transfered to the client yet. In that case try to send a response to the client containing the 500 error.
Maybe create a new issue for that?